src/utils/network.h

```diff
@@ -34,4 +34,11 @@
 #define PG_VTEP_I_FLAG 0x08000000
 #define PG_VTEP_BE_I_FLAG PG_CPU_TO_BE_32(PG_VTEP_I_FLAG)
 
+struct eth_ipv4_hdr {
+	struct ether_hdr eth;
+	struct ipv4_hdr ip;
+	/* the begin of the message */
+	uint64_t beg_msg;
```

Comment:
well, `uint64_t beg_msg` isn't IP nor it's Ethernet so it should not be on this struct, which can lead to bug
if someone do a sizeof(struct eth_ipv4_hdr).